experimental/veggieseasons/README.md

```diff
@@ -1,42 +0,0 @@
-# Veggie Seasons
```

Comment:
Can you add this file back with a pointer to the new location? This way anyone linked to this directory will see an appropriate redirection notice.

Reply:
Sure, I have updated the README accordingly. PTAL.